docs/contributor/voiceover.rst

```diff
@@ -0,0 +1,166 @@
+.. _voiceovers:
+
+Adding voiceovers to lessons
+=============================
+
+Voiceovers help make Oppia's lessons more accessible to non-native speakers. In addition, they give a touch of personality and introduce another dimension to a lesson.
+
+In this section, you will learn how to:
+
+ * :ref:`Apply to do a voiceover <voiceoverapplication>`
+ * :ref:`Record voiceovers for a lesson <recordvoiceover>`
+
+Don't forget to check out :ref:`these guidelines <guidelines>` for recording a good voiceover.
+
+Watch the video to learn how to contribute voiceovers, or if you prefer reading the instructions instead, keep scrolling down.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/0LabiUz9hp4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+.. _voiceoverapplication:
+
+Applying to do a voiceover
```

Comment:
Applying to become a voice-artist* (here and elsewhere)

Reply:
Done